spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitManagementTemplate.java

```diff
@@ -70,6 +70,20 @@ public RabbitManagementTemplate(Client rabbitClient) {
 
 	/**
 	 * Construct a template using the supplied uri.
+	 * @param uri the uri - must include user info, e.g.
+	 * "http://guest:guest@localhost:15672/api/".
```

Comment:
True; that's what I did for my [CloudFoundry PoC](https://github.com/garyrussell/sandbox/blob/master/repartition/src/main/java/org/springframework/amqp/protonj/RepartitionApplication.java#L120).